### DISCUSSION TITLE
Error: Failed to load config "standard" to extend from.

### DISCUSSION THREAD

**Discussion by BossamyD** [VDI-CodING/codeING-main#35] [S1D0]
<img width="957" alt="eslint_standard" src="https://user-images.githubusercontent.com/80851868/111524020-3ae8ac00-875c-11eb-90a9-9cc47c0322b9.PNG">

Vielen Dank für die Einführung! Ich konnte unter localhost:5000 die "Hallo Welt" Seite vorfinden, jedoch hatte ich zunächst Probleme eslint zu laden und habe es dann mit -g global gesetzt. Dann war das Verbotszeichen (unten rechts) zu erkennen und ich habe lediglich auf "Allow" und nicht auf "Allow all and everywhere" geklickt. War das falsch? Jetzt bekomme ich einen Error mit "standard".

Dankbar für Tipps.

**Comment by Zangarus** [comment]
Das "Allow" ist schon richtig. Dies ist nur die Einstellung deiner IDE (Visual Studio Code), dass es die Erweiterung ESLint nutzt. Das Problem ist hier, dass ESLint nicht richtig installiert ist. Da du es mit -g global installiert hast, sucht es auch in der globalen Umgebung nach der config "standard". Dies ist aber nur lokal bei dir in node_modules installiert.

Da eine globale Installation der Dependencies nicht ratsam ist (vgl. z.B. https://github.com/eslint/eslint/issues/1238#issuecomment-55027684), solltest du versuchen es wieder global zu uninstallieren `npm uninstall -g <package-name>`.
Kannst du mir sagen, was für Probleme auftreten, wenn du es versuchst lokal zu installieren? Gerne wieder die Fehlermeldung mitschicken :)

**Reply by BossamyD** [reply]
Vielen Dank für die Antwort. Ich habe folgendes durchgeführt und ich bekomme denselben Error. Soll ich npm fund versuchen, wie vorgeschlagen? 
<img width="461" alt="eslint_install" src="https://user-images.githubusercontent.com/80851868/111670069-ba878100-8817-11eb-8f27-75d458470705.PNG">

**Reply by BossamyD** [reply]
Das ist der Error in VSC:
<img width="953" alt="eslint_error" src="https://user-images.githubusercontent.com/80851868/111670283-f4588780-8817-11eb-85fd-15aadfcb2d73.PNG">

**Reply by BossamyD** [reply]
Alternativ, kann ich nochmal Schritt 8 von S1D0 wiederholen. Ich weiß nur nicht mehr, welche config-Einstellungen wir da getätigt haben und weiß nicht, wo ich das Video finde. Vielen Dank vorab! :)

**Reply by Zangarus** [reply]
Das Video kannst du [hier](https://1drv.ms/u/s!Apib2MdOJ2WvtowzTb6CDyAlCSZ8xw?e=m7mWw9) herunterladen.

Die `npm WARN` Meldung kannst du ignorieren. Diese sollte keine Probleme verursachen. Morgen laden wir eine verbesserte Anleitung zum Aufsetzen des Projekts in Schriftform hoch und auch eine bessere Videoanleitung. Wenn du Schritt 8 mit diesen wiederholst, sollte sich das Problem lösen lassen.

Ist dies nicht der Fall, melde dich bitte nochmal.

**Reply by Blumi102** [reply]
Es scheint, als hast du eslint mit der Kommando-Zeile von Git installiert. Kann es sein, dass das iwie Probleme verursacht? 
(Ich habe das selbst noch nie versucht, ist nur eine Vermutung.)

**Reply by BossamyD** [reply]
Ich habe das gesamte Set-up mit git bash gemacht. Ich benutze git bash als Alternative zur Windows cmd, da hier die Linux Befehle funktionieren. :) Bisher hatte ich keine Probleme und auch alle anderen Schritte (außer 8) haben bei mir problemlos funktioniert. Nur eslint macht Probleme, was im Bezug auf das Hackathon relevant werden könnte, wenn ich im Team arbeite. 

Vielen Dank vorab für weitere Tipps zum Wiederholen von Schritt 8. Ich warte dann auf die Anleitung :)

**Reply by stan1025** [reply]
@BossamyD ich hab mal etwas recherchiert über diesen Fehler.
Du und über 100 andere haben das Problem auch. Das Problem scheint in einem geänderten Solver-Mechanismus für Plugins in ESLint v6 zu sein. Auf den entsprechenden Github Seiten wird der Issue derzeit noch stark diskutiert, da er wie sich hier im Dojo zeigt, nicht so einfach reproduzierbar zu sein scheint. 

Ein Teil der Diskussionen ist hier drin https://github.com/microsoft/vscode-eslint/issues/696 , wobei es gibt weit über 300 Kommentare dazu.

Ich hab mal ein paar Tests die du machen kannst, ob einer davon dein Problem löst. 

[ ] Schließe alle VS Code Fenster und gehe im Explorer in den Ordner des Repository. Klicke in die freie Fläche unterhalb der Auflistung und öffne das Kontextmenu - > Klicke auf Öffnen mit VS Code. Probiere dann, ob der Fehler noch besteht.

Falls das auch nicht funktioniert, führe folgenden Befehl aus auf dem Root-Verzeichnis

[ ] eslint --fix (siehe https://github.com/microsoft/vscode-eslint/issues/696#issue-460414134)

Falls das auch nicht funktioniert, wäre ein neu aufsetzen nochmal ein versucht wert. 
Ggfs. schaltet sich einer von uns mit dir kurz und wir vereinbaren ein Online-Call zur Klärung.
